Assets/__Scripts/MapEditor/UI/PrecisionStepDisplayController.cs

```diff
@@ -12,17 +12,26 @@ public class PrecisionStepDisplayController : DisableActionsField
     [SerializeField] private Color defaultOutlineColor;
     [SerializeField] private Color selectedOutlineColor;
 
-    private bool firstActive;
+    private bool _firstActive;
```

Comment:
out of curiosity, why was there an underscore added to this field?

Reply:
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md #3 🤷‍♂️
I can change it back not really a big deal

Reply:
nah underscore is better